src/main/java/com/cs56fitnessapp/models/FoodEntry.java

```diff
@@ -0,0 +1,94 @@
+package com.cs56fitnessapp.models;
+
+public class FoodEntry {
+    private double servingSize;
+    private int calories;
+    private TotalFat totalFat;
+    private double cholesterol;
+    private double sodium;
+    private TotalCarb totalCarbs;
+    private double protein;
+    private double vitaminA;
+    private double vitaminC;
+    private double calcium;
+    private double iron;
+    public FoodEntry(double servingSize, int calories, TotalFat totalFat, double cholesterol, double sodium, TotalCarb totalCarbs, double protein, double vitaminA, double vitaminC, double calcium, double iron){
+        this.servingSize = servingSize;
+        this.calories = calories;
+        this.totalFat = totalFat;
+        this.cholesterol = cholesterol;
+        this.sodium = sodium;
+        this.totalCarbs = totalCarbs;
+        this.protein = protein;
+        this.vitaminA = vitaminA;
+        this.vitaminC = vitaminC;
+        this.calcium = calcium;
+        this.iron = iron;
+    }
+    public double getServingSize(){
+        return servingSize;
+    }
+    public void setServingSize(double a){
+        servingSize = a;
+    }
+    public int getCalories(){
+        return calories;
+    }
+    public void setCalories(int b){
+        calories  = b;
+    }
+    public TotalFat getTotalFat(){
+        return totalFat;
+    }
+    public void setTotalFat(TotalFat c){
```

Comment:
Style note. It'a good practice to add a white space before a curly brace of the block like that:

`public void setCalories(int b) {}`

Also a good practice - have your variables/parameters name descriptive, e.g. `int calories` instead of `int b`, so in general do not use a,b,c as variables names. Here it would be good to rename them.